release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1941,73 +1941,73 @@ As of {product-title} {product-version}, the `oc registry login` command no long
 
-* Previously, security changes to Amazon S3 buckets caused the Cloud Credential Operator utility (`ccoctl`) command that is used to create AWS resources during installation (`ccoctl aws create-all`) to fail. With this release, the `ccoctl` utility is updated to reflect the Amazon S3 security changes. (link:https://issues.redhat.com/browse/OCPBUGS-11671[*OCPBUGS-11671*])
+* Previously, security changes to Amazon S3 buckets caused the Cloud Credential Operator utility (`ccoctl`) command that is used to create AWS resources during installation (`ccoctl aws create-all`) to fail. With this release, the `ccoctl` utility is updated to reflect the Amazon S3 security changes. (link:https://issues.redhat.com/browse/OCPBUGS-11671[OCPBUGS-11671])
 
 [discrete]
 [id="ocp-4-14-cluster-version-operator-bug-fixes"]
 ==== Cluster Version Operator
 
 * Previously, the Cluster Version Operator (CVO) did not reconcile `SecurityContextConstraints` resources as expected. The CVO now properly reconciles `SecurityContextConstraints` resources towards the state defined in the release image, reverting any unsupported modifications to them.
 +
-Users who want to upgrade from earlier {product-title} versions and who operate workloads depending on modified system `SecurityContextConstraints` resources must follow the procedure in the link:https://access.redhat.com/solutions/7033949[Knowledge Base article] to make sure their workloads are able to run without modified system `SecurityContextConstraint` resources. (link:https://issues.redhat.com/browse/OCPBUGS-19465[*OCPBUGS-19465*])
+Users who want to upgrade from earlier {product-title} versions and who operate workloads depending on modified system `SecurityContextConstraints` resources must follow the procedure in the link:https://access.redhat.com/solutions/7033949[Knowledge Base article] to make sure their workloads are able to run without modified system `SecurityContextConstraint` resources. (link:https://issues.redhat.com/browse/OCPBUGS-19465[OCPBUGS-19465])
 
-* Previously, the Cluster Version Operator did not prioritize likely targets when determining which conditional update risks to evaluate first. Now for conditional updates to which risks do not apply, these updates are available faster after Cluster Version Operator detection. . (link:https://issues.redhat.com/browse/OCPBUGS-5469[*OCPBUGS-5469*])
+* Previously, the Cluster Version Operator did not prioritize likely targets when determining which conditional update risks to evaluate first. Now for conditional updates to which risks do not apply, these updates are available faster after Cluster Version Operator detection. . (link:https://issues.redhat.com/browse/OCPBUGS-5469[OCPBUGS-5469])
```

Comment:
```suggestion
* Previously, the Cluster Version Operator did not prioritize likely targets when determining which conditional update risks to evaluate first. Now for conditional updates to which risks do not apply, these updates are available faster after Cluster Version Operator detection. (link:https://issues.redhat.com/browse/OCPBUGS-5469[OCPBUGS-5469])
```
out of scope, but there's a extra period here

Reply:
Well spotted. Updated!